src/shared/rollbar.js

```diff
@@ -0,0 +1,10 @@
+import Rollbar from 'rollbar';
+
+const rollbar = new Rollbar({
+  accessToken: process.env.NEXT_PUBLIC_ROLLBAR_CLIENT_TOKEN,
```

Comment:
Likewise here.

Reply:
(although probably best to only require that it be set if `NODE_ENV === 'production'`).